src/soundbase.cpp

```diff
@@ -184,14 +181,14 @@ QString CSoundBase::SetDev ( const QString strDevName )
             // to be able to access the ASIO driver setup for changing, e.g., the sample rate, we
             // offer the user under Windows that we open the driver setups of all registered
             // ASIO drivers
-            sErrorMessage = sErrorMessage + "<br/>" + tr ( "Do you want to open the ASIO driver setups?" );
+            sErrorMessage = sErrorMessage + "<br/>" + tr ( "Do you want to open the ASIO driver setup to change driver settings?" );
 
             if ( QMessageBox::Yes == QMessageBox::information ( nullptr, APP_NAME, sErrorMessage, QMessageBox::Yes | QMessageBox::No ) )
             {
                 LoadAndInitializeFirstValidDriver ( true );
```

Comment:
What if this fixes the problem?  We still call `thow CGenErr`?

Reply:
I think I mentioned something like this somewhere else (talked about strStat as far as I know). 

Probably worth another issue & more investigation.

Reply:
See https://github.com/jamulussoftware/jamulus/issues/1812